web_tree_image_tooltip/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Therp BV (<http://therp.nl>).
+# Copyright 2015 Leonardo Donelli @ MONKSoftware
+# Copyright 2013 Marcel van der Boom <marcel@hsdev.com>
+# Copyright 2016 - TODAY Serpent Consulting Services Pvt. Ltd.
+#                            (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Show images in tree views via tooltip',
+    'version': '12.0.1.0.0',
+    'author': "Therp BV, "
+              "MONK Software, "
+              "Odoo Community Association (OCA), "
+              "Serpent Consulting Services Pvt. Ltd.",
+    'website': 'https://github.com/OCA/Web',
```

Comment:
Use lower case characters for the repository url so it's consistent with all other modules. https://github.com/OCA/Web -> https://github.com/OCA/web